katello/katello-debug.sh

```diff
@@ -57,6 +57,9 @@ add_files /var/log/tomcat*/manager*.log*
 add_files /etc/candlepin/candlepin.conf
 add_files /etc/tomcat*/server.xml
 
+# RHSM
+add_files /var/log/rhsm/rhsm.log
```

Comment:
Probably should be `/var/log/rhsm/rhsm*.log` or `/var/log/rhsm/*.log`. Some people rotate these :-)